src/bentoml/bentos.py

```diff
@@ -4,9 +4,11 @@
 
 from __future__ import annotations
 
+import shutil
 import typing as t
 import logging
 from typing import TYPE_CHECKING
+from subprocess import Popen
```

Comment:
nit:
```suggestion
import subprocess
```

Reply:
Thanks, Aaron. For my notes, could you help me understand why this change is better? My understanding was that `from <x> import <y>` would have better performance than `import <x>`. Or is it because the improvement there would be trivial and `import <x>` is cleaner/more readable?

Reply:
Resolving this and moving this question to Slack.